app/views/application_letters/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <%- model_class = ApplicationLetter -%>
 <div class="page-header">
-  <h1><%=t '.title', :default => model_class.model_name.human.pluralize.titleize %></h1>
+  <h1><%=t '.title', :default => model_class.model_name.human(:count => model_class.count).titleize %></h1>
```

Comment:
Well I think it will lead to an bug report, as soon as this page gets used. But I think you can merge it anyway